insights/specs/sos_archive.py

```diff
@@ -10,6 +10,10 @@
 
 class SosSpecs(Specs):
     blkid = simple_file("sos_commands/block/blkid_-c_.dev.null")
+    candlepin_log = first_of([
```

Comment:
I don't think you used the master branch as the base of this PR. The patch of #1142 is also included.